ansible/index.md

```diff
@@ -50,6 +50,17 @@ These are maximums. Power consumption varies greatly based on which USB controll
 - 19ma @ -12V
 - No +5v required
 
+## Panel details
+
+![Panel Overlay](images/ansible_CLEAN.png)
+
+- 4 Trigger outputs (TR 1-4, voltage range, slew rate, sample rate, bit rate?)
```

Comment:
I'm guessing the TR outputs are pure binary (on or off), 0 or 10v?

In that case, bit rate and slew rate might be irrelevant, but sample rate, or maximum toggle rate would be the helpful parameter to include here